File: HackerRank/minimaxsum.py

def miniMaxSum(arr):
    ma = 0
    mi = 10**9
    n = len(arr)
    for i in range(n):
        result = 0
        for j in range(n):
            result += arr[j]

        result -= arr[i]
        if ma < result:
            ma = result

        if mi > result:
            mi = result

    print(mi, ma)

if __name__ == '__main__':
    arr = [1, 2, 3, 4, 5]
    miniMaxSum(arr)

File: HackerRank/minimaxsum.c
#include <stdio.h>

void miniMaxSum(int arr_count, int* arr) {
	long int sum = 0;
	long int min = arr[0];
	long int max = arr[0];
	for (int i = 0; i < arr_count; i++) {
		sum += arr[i];
		if (arr[i] < min) {
			min = arr[i];
		}
		if (arr[i] > max) {
			max = arr[i];
		}
	}
	printf("%ld %ld", sum - max, sum - min);
}

int main (  ){
	int arr[5] = {1, 2, 3, 4, 5};
	miniMaxSum(5, arr);
	return 0;
	
}
